README.md

```diff
@@ -344,8 +344,35 @@ For a comparison of some of these, see this
 
 ## Storage Backends
 
-In order to not download or scan any previously scanned sources again, the _scanner_ can use a storage backend to store
-scan results for later reuse.
+In order to not download or scan any previously scanned sources again or to reuse scan results generated via other
+services, the _scanner_ can be configured to use so-called storage backends. Before processing a package, it checks
+whether compatible scan results are already available in one of the storages declared; if this is the case, they
+are downloaded and reused. Otherwise, the package is scanned now. Afterwards, the new results can be put into a
+storage for later reuse.
+
+It is possible to configure multiple storages to read scan results from and to write new results to independently.
```

Comment:
Let's make this a bit shorter for clarity: "It is possible to configure multiple storages to read scan results from or to write scan results to."